### PR TITLE
feat(Emails) Add aftersend action to PHPMailer e-mail sending

### PR DESCRIPTION
**Pull request by Luke1982**

### CODE REVIEW

File: modules/Emails/Emails.php [hunk 1]
```diff
@@ -298,6 +298,9 @@ public static function sendEMail(
 			$mail->AddReplyTo($HELPDESK_SUPPORT_EMAIL_REPLY_ID);
 		}
 		$mail_status = MailSend($mail);
+
+		cbEventHandler::do_action('corebos.email.aftersend', $mail);
```
Comment (joebordes) [inline modules/Emails/Emails.php:302]: add the $mail_status to the parameters that the event receives and I will accept. LGTM

Comment (Luke1982) [reply modules/Emails/Emails.php:302]: Can we do that by just adding a parameter to the `do_action` call or do I need to wrap it in an array?

Comment (joebordes) [reply modules/Emails/Emails.php:302]: has to be wrapped in an array

Comment (Luke1982) [reply modules/Emails/Emails.php]: Done. The receiving function should unwrap that array right? That doesn't happen automagically?

**Comment by joebordes** [comment]
Thanks!!

File: modules/Emails/Emails.php (hunk 1 above)
Comment (joebordes) [reply modules/Emails/Emails.php]: Yes, correct

**Pull request merged**